production/helm/promtail/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: "v1"
 name: promtail
-version: 0.19.2
+version: 0.19.3
```

Comment:
You also need to bump the loki-stack since promtail belong to it.

Reply:
Alright, bumped it as well :-)